tests/conftest.py

```diff
@@ -1,9 +1,172 @@
 """Fixtures common across tests."""
+import pickle
+from typing import Callable, List
 
+import gym
 import pytest
 import torch
+from stable_baselines3 import PPO
+from stable_baselines3.common.policies import BasePolicy
+from stable_baselines3.common.vec_env import DummyVecEnv, VecNormalize
 
+from imitation.data import rollout
+from imitation.data.types import TrajectoryWithRew
+from imitation.data.wrappers import RolloutInfoWrapper
+from imitation.policies.base import FeedForward32Policy, NormalizeFeaturesExtractor
 from imitation.util import logger
+from imitation.util.networks import RunningNorm
+
+CARTPOLE_ENV_NAME = "CartPole-v1"
+
+
+def load_or_train_ppo(
```

Comment:
How does this interact with `pytest-xdist`? Are we using a lock file somewhere, or did you work out a better solution?

Reply:
Good catch! I forgot about it, now added a lock file so the workers are waiting for each other to train the model.